.github/workflows/notify-triager.yml

```diff
@@ -0,0 +1,69 @@
+name: Notify Triagers
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+
+jobs:
+  Notify-triagers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
```

Comment:
This step should also be added I believe to minimize running of actions on bot PRs. https://github.com/asyncapi/.github/blob/09d72fb0774103e4fd36b81cfac1d15f8a3fbba7/.github/workflows/if-nodejs-pr-testing.yml#L19-L35

Reply:
@akshatnema Makes sense, I think we should. WDYT?

Reply:
Yepp makes sense 👍